src/SearchResults/generate-hierarchical-tree.js

```diff
@@ -31,7 +29,7 @@ function generateTrees(state) {
       results = hierarchicalFacetResult.slice(hierarchicalRootPath.split(hierarchicalSeparator).length);
     }
 
-    return reduce(results, generateTreeFn, {
+    return results.reduce(generateTreeFn, {
```

Comment:
maybe an object

Reply:
Nope it's an array, worked on that piece 👍